README.md

```
@@ -0,0 +1,52 @@
+- Store and retrieve text embeddings in PostgreSQL using Entity Framework Core and pgvector.
+- Generate embeddings via Ollama API.
+- REST endpoints for:
+  - Adding new texts and their embeddings.
+  - Searching for similar texts using vector similarity.
+  - Listing stored texts.
+
+## Usage
+
+Simply run:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)